sdk/oracle/Azure.ResourceManager.OracleDatabase/CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 # Release History
 
-## 1.0.0-beta.1 (2024-06-18)
+## 1.0.0 (2024-06-28)
+
+### Features Added
+
+This is the first stable release.
 
 ### General New Features
```

Comment:
```suggestion
```

This isn't a standard header and shouldn't be used.   This should fall under the features added.

Reply:
Fixed